components/profile_popover/profile_popover.jsx

```diff
@@ -440,8 +440,10 @@ class ProfilePopover extends React.PureComponent {
         if (this.props.hasMention) {
             title = <a onClick={this.handleMentionKeyClick}>{title}</a>;
         }
+
+        const escapedUsername = this.props.user.username.replace('.', '-');
         title = (
-            <span>
+            <span id={`userPopoverUserName-${escapedUsername}`}>
```

Comment:
Per recent agreement, we'll be using `data-testid` especially for this case where dynamic ID can be conformant HTML id attribute